package.json

```diff
@@ -26,7 +26,7 @@
     "stats:build": "gulp stats",
     "stats:save": "gulp stats:save",
     "satisfied": "satisfied --skip-invalid",
-    "pretest": "yarn satisfied",
+    "pretest": "yarn satisfied && yarn syncpack list-mismatches",
```

Comment:
could you, please, suggest what is the approx time cost of this change? I mean, what is the relative change of time necessary to pretest commit changes?

Reply:
`pretest` will be run in CI, too. My idea is too check dependencies there, example throw will be:
```
$ syncpack list-mismatches
lodash ^4.17.11, ^4.17.10
error Command failed with exit code 1.
```